db/schema.rb

```diff
@@ -126,6 +126,7 @@
     t.string "email"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "service", default: 0
```

Comment:
I think this should be a null false but with no default value. The migration could:

1. Add the column with a default value of `check` 
2. Change the column default to nil